pywhat/what.py

```diff
@@ -126,7 +126,7 @@ def what_is_this(self, text: str) -> dict:
         """
         Returns a Python dictionary of everything that has been identified
         """
-        return self.id.identify(text)
+        return self.id.identify(text, only_text=False)

```

Comment:
Am I reading this correctly? There is no way to set `only_text=True` in this function? 

This means:
https://github.com/bee-san/pyWhat/blob/a4954ef6c3531c5ce87169de13b829d1fea68746/pywhat/what.py#L114-L118

For CLI input you never have only text? it will always check for a file? Might be worth making an argument for this?